clusterDeployer.py

```diff
@@ -553,11 +554,13 @@ def wait_for_workers(self) -> None:
         lh = host.LocalHost()
         bf_workers = [x for x in self._cc.workers if x.kind == "bf"]
         connections: Dict[str, host.Host] = {}
-        while True:
+        for try_count in itertools.count(0):
             workers = [w.name for w in self._cc.workers]
-            if all(self.client().is_ready(w) for w in workers):
+            ready_workers = (self.client().is_ready(w) for w in workers)
```

Comment:
the generator expression (`(...)`) will be evaluated twice.

That is not a concern for performance, but in general, it seems ugly to call multiple times into non-trivial code (what is `self.cliend().is_ready(w)` doing?) and assume that they calls give the same results every time. Evaluate the result only once by using `[]`.

Reply:
Ack.